src/Exceptions/HttpRequestTimeout.php

```diff
@@ -8,4 +8,3 @@ class HttpRequestTimeout extends HttpException {
     public const DESCRIPTION = "Indicates that the server did not receive a complete request message within the time that it was prepared to wait.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/408";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces